pkg/apis/fleet/v1alpha1/types.go

```diff
@@ -248,6 +250,57 @@ type PodSecurityPolicy struct {
 	ValidationFailureAction string `json:"validationFailureAction,omitempty"`
 }
 
+// BackupConfig defines the configuration for backups.
+type BackupConfig struct {
+	// Chart defines the helm chart configuration of the backup engine.
+	// The default value is:
+	//
+	// chart:
+	//   repository: https://vmware-tanzu.github.io/helm-charts
+	//   name: velero
+	//   version: 5.0.2
+	//
+	// +optional
+	Chart *ChartConfig `json:"chart,omitempty"`
+
+	// Storage provides details on where the backup data should be stored.
+	Storage BackupStorage `json:"storage"`
+
+	// ExtraArgs provides the extra chart values for the backup engine chart.
+	// For example, use the following configuration to change the image tag or pull policy:
+	//
+	// extraArgs:
+	//   image:
+	//     repository: velero/velero
+	//     tag: v1.11.1
+	//     pullPolicy: IfNotPresent
+	//
+	// +optional
+	ExtraArgs apiextensionsv1.JSON `json:"extraArgs,omitempty"`
+}
+
+type BackupStorage struct {
+	// Location specifies where the backup data will be stored.
+	Location BackupStorageLocation `json:"location"`
+
+	// Credentials refers to the Kubernetes secret containing the AccessKeyID and SecretAccessKey
+	// required to access the backup storage location. The secret might, for example,
+	// contain fields such as `accessKeyID` and `secretAccessKey` to store the credentials.
```

Comment:
kind of remember we use acessKey and secretKey previously for thanos

Reply:
ok